content/en/cosign/sign.md

```diff
@@ -161,19 +161,112 @@ The upload is skipped by using the `--upload=false` flag (default true). To capt
 $ cosign sign --key key.pem --upload=false --output-signature demo.sig --output-certificate demo.crt user/demo
 ```
 
-## Generate the signature payload (to sign with another tool)
+## Generate the Signature Payload With `cosign` (to sign with another tool)
 
-The json payload is printed to stdout:
+You can also use other tools for signing - not just `cosign`. This section will provide examples of how to sign with tools other than `cosign`.
+
+### GCP KMS with `gcloud`
+
+To sign with `gcloud kms` , first use `cosign generate` to generate the payload and dump it into a JSON file:
+
+```shell
+$ cosign generate us-central1-docker.pkg.dev/dlorenc-vmtest2/test/taskrun > payload.json
+```
+
+Sign the payload with `gcloud kms`:
+
+```shell
+$ gcloud kms asymmetric-sign --digest-algorithm=sha256 --input-file=payload.json --signature-file=gcpkms.sig --key=foo --keyring=foo --version=1 --location=us-central
```

Comment:
Can you break this onto more than one line so it's easier for the reader (that is, so that they don't need to scroll horizontally)